src/mscorlib/corefx/System/Globalization/SortKey.cs

```diff
@@ -52,26 +52,6 @@ internal SortKey(String localeName, String str, CompareOptions options, byte[] k
             _options    = options;
             _string   = str;
         }
-
-        [OnSerializing]
```

Comment:
Looks like yes it was previously stripping it. I'll open a bug in corefx to add tests for serializing those.

---
In reply to: [99196301](https://github.com/dotnet/coreclr/pull/9269#discussion_r99196301) [](ancestors = 99196301,99095564)